addons/sourcemod/scripting/surftimer/misc.sp

```diff
@@ -1,11 +1,16 @@
-public void SetBotQuota()
+void setBotQuota()
```

Comment:
Don't change the name of the function, please

Reply:
I did just change it back :) sorry

Reply:
```cpp
public void SetBotQuota()
```
instead of
```cpp
public void setBotQuota()
```
please